include/iGenVar.hpp

```diff
@@ -27,11 +27,13 @@ struct cmd_arguments
 // SV specifications:
     /* -k */ int32_t min_var_length = 30;
     /* -l */ int32_t max_var_length = 100000;
-    /* -m */ int32_t max_tol_inserted_length = 5;
+    /* -m */ int32_t max_tol_inserted_length = 50;
+    /* -e */ int32_t max_tol_deleted_length = 50;
     /* -n */ int32_t max_overlap = 10;
     /* -q */ int32_t min_qual = 1;
 // Clustering specifications:
-    /* -w */ double hierarchical_clustering_cutoff = 100;
+    /* -p */ int32_t partition_max_distance = 1000;
```

Comment:
If the user must pass non-negative values, just use an unsigned type. This automatically puts `(unsigned 32 bit integer)` to the help screen and prints a helpful error message if a user passes a negative value nonetheless.
```suggestion
    /* -p */ uint32_t partition_max_distance = 1000;
```

Reply:
I opened an issue for that: https://github.com/seqan/iGenVar/issues/161